layers/parameter_validation_utils.cpp

```diff
@@ -8684,6 +8684,14 @@ bool StatelessValidation::ValidateDeviceImageMemoryRequirements(VkDevice device,
                              "%s(): pInfo->pCreateInfo->pNext chain contains VkImageDrmFormatModifierExplicitCreateInfoEXT.",
                              func_name);
         }
+
+        if ((pInfo->pCreateInfo->flags & VK_IMAGE_CREATE_DISJOINT_BIT) != 0) {
+            if (FormatIsMultiplane(pInfo->pCreateInfo->format) && (pInfo->planeAspect == VK_IMAGE_ASPECT_NONE_KHR)) {
+                skip |=
+                    LogError(device, "VUID-VkDeviceImageMemoryRequirementsKHR-pCreateInfo-06417",
+                             "%s(): Can not use VK_IMAGE_ASPECT_NONE_KHR with a multiplanar format with disjoint flag.", func_name);
```

Comment:
I think this reads a bit easier.

```suggestion
                             "%s(): Must not specify VK_IMAGE_ASPECT_NONE_KHR with a multi-planar format and disjoint flag", func_name);
```

Reply:
I realize we don't do it consistently but I think it would be good practice to use `Must` in the error messages if the VUID does. Since it's part of the vulkan specification [normative terminology](https://registry.khronos.org/vulkan/specs/1.3/html/chap2.html#introduction-normative-terminology)

Reply:
agree 100%, good catch